tools/run-in-simulator.sh

```diff
@@ -21,6 +21,8 @@ xcrun simctl launch --console-pty "$id" "$bundleId"
 # with F_FULLFSYNC, and any unflushed writes are lost entirely if the simulator
 # is shut down. As a result, we need to wait for the test results file to
 # actually appear on disk before continuing.
+# Wait 5 seconds before checking the first time
```

Comment:
Is this from some local debugging or something?

Reply:
On one of my runs, jenkins failed for the iossimulator test because the output file was empty, even though the tests were successful. In the log, the test returned immediately and killed the simulator on the first check of the results file, which seems to me that the results file had been created but hadn't finished writing since the file size was 0.

I added the initial 5 second wait before checking for the file to help prevent this from happening in the future.